lib/russian_central_bank_safe.rb

```diff
@@ -0,0 +1,24 @@
+class RussianCentralBankSafe < Money::Bank::RussianCentralBank
+  def update_rates(date = Date.today)
+      @mutex.synchronize{
+        begin
+          update_parsed_rates exchange_rates(date)
+          @rates_updated_at = Time.now
+          @rates_updated_on = date
+          @rates
+        rescue Exception => e
+          begin
+            last_db_rates = ExchangeRate.last
```

Comment:
@shir We try to update rates from bank
If HTTP 302 is returned then we try to take values from our DB.
But when server starts there is no connection to DB when initializing money.rb and setting default bank and rates, in that case we parse values from the yml file